src/odemis/gui/comp/legend.py

```diff
@@ -262,6 +262,8 @@ def __init__(self, parent, wid=wx.ID_ANY, pos=(0, 0), size=wx.DefaultSize,
         self._max_tick_width = 32  # Largest pixel width of any label in use
         self._tick_spacing = 120 if orientation == wx.HORIZONTAL else 80
         self._unit = None
+        self._lo_ellipsis = False
```

Comment:
Add a comment to indicate what they mean. You could also add a docstring to the class and explain how the two attributes .lo_ellipsis and .hi_ellipsis are to be used.